### PR TITLE
Fixed link

### PR DESCRIPTION
**Pull request by BrandonDoesDev**
Just fixed a broken link on readme

Don't worry, I'll contribute to the actual code soon ;)

### CODE REVIEW

**Comment by ftqo** [comment]
I believe a set of parentheses need to be removed at the end of the link.

File: README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ func main() {
 
 ### Full Ping Pong Example
 
-This example can also be found [here]https://github.com/disgoorg/disgo/blob/development/_examples/ping_pong/example.go()
+This example can also be found [here](https://github.com/disgoorg/disgo/blob/development/_examples/ping_pong/example.go())
```
Comment (TopiSenpai) [inline README.md:99]:
```suggestion
This example can also be found [here](https://github.com/disgoorg/disgo/blob/development/_examples/ping_pong/example.go)
```

**Review by TopiSenpai — CHANGES_REQUESTED** [review]
Nice catch

**Comment by TopiSenpai** [comment]
if you want this merged you should apply my suggestion btw

**Review by TopiSenpai — APPROVED** [review]

**Pull request merged**